pollux/lib/core/src/main/scala/io/iohk/atala/pollux/core/service/verification/VcVerificationService.scala

```diff
@@ -6,17 +6,18 @@ trait VcVerificationService {
   def verify(request: List[VcVerificationRequest]): IO[VcVerificationServiceError, List[VcVerificationResult]]
 }
 
+sealed trait VcVerificationParameter
+
+case class AudienceParameter(aud: String) extends VcVerificationParameter
+
 final case class VcVerificationRequest(
     credential: String,
-    verifications: List[VcVerification]
+    verification: VcVerification,
```

Comment:
Why not revert to what you did before, that is, passing a list of verifications to the service `verify` method? This allows one to make a single service call to check, e.g. 10 properties of a VC, instead of making 10 separate calls. I find it more efficient and elegant...

Reply:
I am not passing a list of VcVerificationRequest in a single call. It simplifies the logic.